### PR TITLE
Update user-submission.md

### PR DESCRIPTION
**Pull request by andypunt**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/user-submission.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ the following settings:
 
 You can configure third-party message reporting tools to send reported messages to the custom mailbox. The only requirement is that the original message is included as an attachment in the message that's sent to the custom mailbox (don't just forward the original message to the custom mailbox).
 
-The message formatting requirements are described in the next section.
+The message formatting requirements are described in the next section. The formatting is optional, but if it does not follow the prexscribed format, the reports will always be submitted as phish.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/user-submission.md]:
```suggestion
The message formatting requirements are described in the next section. The formatting is optional, but if it does not follow the prescribed format, the reports will always be submitted as phish.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Pull request merged**